tests/filter-tests.js

```diff
@@ -148,6 +148,7 @@ describe('broccoli-asset-rev', function() {
     var node = new AssetRewrite(sourcePath + '/input', {
       replaceExtensions: ['js'],
       assetMap: {
+        'images/some-image.jpg': 'images/some-image-1a2b3c4e.jpg',
         'the.map' : 'the-other-map',
         'http://absolute.com/source.map' : 'http://cdn.absolute.com/other-map'
```

Comment:
@rickharrison does this line make sense in the real world? Is it a real situation where you have an absolute  sourcemap and you want to replace by a different absolute sourcemap? This is the only failing test (since the `prepend` is set, my algorithm generates `https://cloudfront.net/http://cdn.absolute.com/other-map`), but I don't see much sense on this operation at all.